.github/ISSUE_TEMPLATE/0-bugissue.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug Issue
+about: Use this template for reporting a bug
+---
+
+## System information
+- CPU information(`cat /proc/cpuinfo | grep "model name" | uniq`):
+- GPU information(`lspci -nn | grep -E 'VGA|isplay`):
```

Comment:
Deliberately remove a letter to expand the possibility of matching?

Reply:
Yes, ignoring 'D'isplay to make sure case insensitive.